diff --git a/Modules/ThirdParty/KWSys/src/CMakeLists.txt b/Modules/ThirdParty/KWSys/src/CMakeLists.txt
index 75b8ccd..5be8387 100644
--- a/Modules/ThirdParty/KWSys/src/CMakeLists.txt
+++ b/Modules/ThirdParty/KWSys/src/CMakeLists.txt
@@ -18,7 +18,7 @@ set(KWSYS_INSTALL_DOC_DIR ${ITK_INSTALL_DOC_DIR})
 set(KWSYS_INSTALL_EXPORT_NAME ${ITKKWSys-targets})
 set(KWSYS_INSTALL_COMPONENT_NAME_RUNTIME RuntimeLibraries)
 set(KWSYS_INSTALL_COMPONENT_NAME_DEVELOPMENT Development)
-set(KWSYS_PROPERTIES_CXX MACOSX_RPATH 1)
+#set(KWSYS_PROPERTIES_CXX MACOSX_RPATH 1)
 
 add_subdirectory(KWSys)
 itk_module_target(${KWSYS_NAMESPACE} NO_INSTALL)
