diff --git a/qwtconfig.pri b/qwtconfig.pri
index 21f7965..e216a65 100644
--- a/qwtconfig.pri
+++ b/qwtconfig.pri
@@ -12,6 +12,14 @@ QWT_VER_MIN      = 2
 QWT_VER_PAT      = 0
 QWT_VERSION      = $${QWT_VER_MAJ}.$${QWT_VER_MIN}.$${QWT_VER_PAT}
 
+win32 {
+    CONFIG += no_batch
+}
+
+win64 {
+    CONFIG += no_batch
+}
+
 ######################################################################
 # Install paths
 ######################################################################
@@ -72,7 +80,7 @@ QWT_INSTALL_FEATURES  = $${QWT_INSTALL_PREFIX}/features
 # it will be a static library.
 ######################################################################
 
-QWT_CONFIG           += QwtDll
+QWT_CONFIG           -= QwtDll
 
 ######################################################################
 # QwtPlot enables all classes, that are needed to use the QwtPlot 
@@ -93,7 +101,7 @@ QWT_CONFIG     += QwtWidgets
 # export a plot to a SVG document
 ######################################################################
 
-QWT_CONFIG     += QwtSvg
+QWT_CONFIG     -= QwtSvg
 
 ######################################################################
 # If you want to use a OpenGL plot canvas
@@ -110,7 +118,7 @@ QWT_CONFIG     += QwtOpenGL
 # to your qmake project file.
 ######################################################################
 
-QWT_CONFIG     += QwtMathML
+QWT_CONFIG     -= QwtMathML
 
 ######################################################################
 # If you want to build the Qwt designer plugin, 
@@ -118,7 +126,7 @@ QWT_CONFIG     += QwtMathML
 # Otherwise you have to build it from the designer directory.
 ######################################################################
 
-QWT_CONFIG     += QwtDesigner
+QWT_CONFIG     -= QwtDesigner
 
 ######################################################################
 # Compile all Qwt classes into the designer plugin instead
