diff --git a/qwt/qwtconfig.pri b/qwt/qwtconfig.pri
index a54a905b..ad300722 100644
--- a/qwt/qwtconfig.pri
+++ b/qwt/qwtconfig.pri
@@ -72,7 +72,7 @@ QWT_INSTALL_FEATURES  = $${QWT_INSTALL_PREFIX}/features
 # it will be a static library.
 ######################################################################
 
-QWT_CONFIG           += QwtDll
+QWT_CONFIG           -= QwtDll
 
 ######################################################################
 # QwtPlot enables all classes, that are needed to use the QwtPlot 
@@ -93,7 +93,7 @@ QWT_CONFIG     += QwtWidgets
 # export a plot to a SVG document
 ######################################################################
 
-QWT_CONFIG     += QwtSvg
+QWT_CONFIG     -= QwtSvg
 
 ######################################################################
 # If you want to use a OpenGL plot canvas
@@ -110,7 +110,7 @@ QWT_CONFIG     += QwtOpenGL
 # to your qmake project file.
 ######################################################################
 
-QWT_CONFIG     += QwtMathML
+QWT_CONFIG     -= QwtMathML
 
 ######################################################################
 # If you want to build the Qwt designer plugin, 
@@ -118,7 +118,7 @@ QWT_CONFIG     += QwtMathML
 # Otherwise you have to build it from the designer directory.
 ######################################################################
 
-QWT_CONFIG     += QwtDesigner
+QWT_CONFIG     -= QwtDesigner
 
 ######################################################################
 # Compile all Qwt classes into the designer plugin instead
@@ -141,7 +141,7 @@ win32 {
 # Otherwise you have to build them from the examples directory.
 ######################################################################
 
-QWT_CONFIG     += QwtExamples
+QWT_CONFIG     -= QwtExamples
 
 ######################################################################
 # The playground is primarily intended for the Qwt development 
@@ -159,7 +159,7 @@ QWT_CONFIG     += QwtPlayground
 # Otherwise you have to build them from the tests directory.
 ######################################################################
 
-QWT_CONFIG     += QwtTests
+QWT_CONFIG     -= QwtTests
 
 ######################################################################
 # When Qt has been built as framework qmake wants 
